starter-code/pizza.js

```diff
@@ -1 +1,104 @@
 // Write your Pizza Builder JavaScript in this file.
+$(function () {
+    var totalPrice = 13;
```

Comment:
It's good to hold the logic in variables :)
Try holding it in some data structure, such as an object:

`
var prices = 
{
pep :  1,
mush: 1
}
`